### DISCUSSION TITLE
How to switch Wi-Fi networks?

### DISCUSSION THREAD

**Discussion by sethhammett** [koreader/koreader#7686] [Q&A] [answered]
How can I choose which Wi-Fi network to connect to? Currently when I turn on Wi-Fi it just auto-connects to my last saved network and I don't see an option to connect to a different network.

**Comment by NiLuJe** [comment, marked as answer]
Long-press the `Wi-Fi connection` menu entry.

**Reply by tjaderxyz** [reply]
Maybe the UI should make this discoverable somehow. Perhaps "..." in the entry or some icon? The same could apply to the setting names in the bottom menu, I only recently discovered I could long press them to open a help box.

**Reply by NiLuJe** [reply]
The Wi-Fi entry is admittedly a problem that I'm not quite sure how to solve.

The rest is handled by the Quickstart guide ;).

**Reply by dcruzbosco** [reply]
Apparently this doesn't work anymore. Long-pressing just enables/disables the Wi-Fi connection.

**Reply by NiLuJe** [reply]
On which device did you try?

**Reply by dcruzbosco** [reply]
Kobo Libra H2O. I'm on the latest release 2022.07.

**Reply by NiLuJe** [reply]
Huh. Still works over here (obviously, you'll want to disable "Dismiss the scan window on connect" first, because that's slightly counter productive for this use-case ;p).

**Reply by dcruzbosco** [reply]
Okay. Yeah, works now. Thanks a lot! :)

**Reply by mvsolves** [reply]
> "Dismiss the scan window on connect"

Where can I find this setting? I am having the same issue as original poster

**Reply by NiLuJe** [reply]
At the bottom of the Network menu. It's only available on platforms where we have a scan window to begin with, though.

**Reply by mvsolves** [reply]
I currently have Kindle PW3, using KOReader v2022.10 and I I cannot change the wifi. It only toggles it. Long holding wifi connection doesn't do anything. Is there anything else I can do to fix this?

**Reply by NiLuJe** [reply]
Because Kindle is not a platform where we control WiFi ourselves, this is deferred to the stock system ;). That makes it toggle only, and it behaves the exact same way as a toggle in the stock system would.
